base.py

```diff
@@ -479,89 +501,14 @@ def _check_modal_window_visibility(self):
         return not windows
 
     def remove_user(self):
-        does_username_exist = self.check_with_retries(
-            self.does_username_exist,
-            self.test_username
-        )
-        assert does_username_exist
-
-        options_btn = self.browser.driver.find_elements_by_css_selector(
-            '.sidebar__toolbar-button.rc-tooltip.rc-tooltip--down.js-button')
-
-        assert len(options_btn)
-
-        self.browser.driver.execute_script('arguments[0].click();',
-                                           options_btn[-1])
-
-        administration_btn = self.browser.find_by_css('.rc-popover__item-text')
-        administration_btn.click()
-
-        users_btn = self.browser.driver.find_elements_by_css_selector(
-            'a.sidebar-item__link[aria-label="Users"]')
-
-        assert len(users_btn)
-
-        self.browser.driver.execute_script("arguments[0].click();",
-                                           users_btn[0])
-
-        selected_user = self.browser.find_by_xpath(
-            '//td[@class="border-component-color"][text()="{0}"]'.
-            format(self.test_username))
-
-        assert len(selected_user)
-
-        selected_user.first.click()
-
-        try:
-            delete_btn = self.find_by_xpath(
-                '//button[@class="js-action rc-user-info-action__item"]'
-                '[text()="Delete"]'
-            )
-
-            assert len(delete_btn)
-
-        except AssertionError:
-            more_btn = self.find_by_css(
-                'button.rc-tooltip.rc-room-actions__button.js-more'
-                '[aria-label="More"]'
-            )
-
-            assert len(more_btn)
-
-            more_btn.first.click()
-
-            delete_btn = self.find_by_xpath(
-                '//li[@class="rc-popover__item js-action"]'
-                '/span[text()="Delete"]'
-            )
-
-            assert len(delete_btn)
-
-        delete_btn.first.click()
-
-        confirm_btn = self.find_by_css('input[value="Yes, delete it!"]')
-
-        assert len(confirm_btn)
-
-        confirm_btn.first.click()
-
-        WebDriverWait(self.browser.driver, 10).until(
-            lambda _: self._check_modal_window_visibility())
-
-        close_btn = self.browser.driver.find_elements_by_css_selector(
-            'button[data-action="close"]')
-
-        assert len(close_btn)
-
-        self.browser.driver.execute_script('arguments[0].click();',
-                                           close_btn[0])
-
-        does_username_exist = self.check_with_retries(
-            self.does_username_exist,
-            self.test_username,
-            expected_res=False
-        )
-        assert not does_username_exist
+        if not self.test_user_id:
+            response = self.rocket.users_list().json()
+            self.test_user_id = [
+                i.get('_id', None) for i in response['users']
```

Comment:
Dict `get` method already returns `NoneType`. It's not necessary to set second argument there.